static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx

```diff
@@ -238,7 +257,21 @@ const PresortedKeyValueList = ({data}: {data: KeyValueListData}) => (
   <KeyValueList shouldSort={false} data={data} />
 );
 
-const makeTransactionNameRow = (event: Event) => makeRow(t('Transaction'), event.title);
+const makeTransactionNameRow = (event: Event, orgSlug: string) => {
+  const to = {
+    pathname: normalizeUrl(`/organizations/${orgSlug}/performance/summary/`),
+    query: {
+      project: event.projectID,
+      transaction: event.title,
+    },
+  };
```

Comment:
We have a way to do this actually
https://github.com/getsentry/sentry/blob/29b76b87c07c6e70f44a52dd2bfd7cf3a6fff928/static/app/views/performance/table.tsx#L206
I'd probably opt to do it that way, and for query you can just make it blank `query: {query: ''}`
which should leave it to default

Reply:
Awesome, thanks! Originally went looking for something like this and couldn't find it. Replaced to use it :+1: